artifactor/plugins/reporter.py

```diff
@@ -37,6 +39,11 @@
     '_duration': 0
 }
 
+URL = re.compile(
```

Comment:
I think it definitely needs a comment to explain what it's for, and if the link seems durable, then linking to the source would likely be valuable to the lucky person that wants to debug this.